.github/CODEOWNERS

```diff
@@ -68,7 +69,6 @@
 /core/templates/services/UpgradedServices.ts @srijanreddy98 @ashutoshc8101
 /core/templates/hybrid-router-module-provider*.ts @srijanreddy98 @ashutoshc8101
 
-
```

Comment:
We are expecting two newlines between project sections. Is this an expected change?

Reply:
Done by mistake, fixed.